mapper/pojo-base/src/main/java/org/hibernate/search/mapper/pojo/massindexing/impl/PojoMassIndexingDelegatingFailureHandler.java

```diff
@@ -39,4 +40,9 @@ public void handle(MassIndexingEntityFailureContext context) {
 		}
 		delegate.handle( builder.build() );
 	}
+
+	@Override
+	public long failureFloodingThreshold() {
+		return FAILURE_FLOODING_THRESHOLD;
```

Comment:
I think it would be better to call `delegate.failureFloodingThreshold()` here, and to implement the default of `100` in `LogFailureHandler`. I think I mentioned this in my comment?

Reply:
hmm ... that would mean that we need to push the `failureFloodingThreshold()` to `org.hibernate.search.engine.reporting.FailureHandler`. But that is a more global handler and not mass indexer specific ... I'll keep it as is, and we can return to this on Monday 😃

Reply:
Yes you would need to *also* have that method on `org.hibernate.search.engine.reporting.FailureHandler`.
That makes sense because `FailureHandler` is the default failure handler in mass indexing.

Reply:
Ohh ok. What concerned me was that this `org.hibernate.search.engine.reporting.FailureHandler` is also used for those `IndexingPlanSynchronizationStrategy`ies, but the threshold is only applicable to mass-indexing (at this moment). Maybe for `org.hibernate.search.engine.reporting.FailureHandler`, we call it `massIndexingFailureFloodingThreshold()` or something along those lines 😃

Reply:
If you want to, but I'll be fine with just `failureFloodingThreshold()`. We can always expand the principle of failure flooding protection to other areas later if it makes sense. For `IndexingPlanSynchronizationStrategy`, it doesn't make sense, really, since there's only ever one failure being reported in the end.

Reply:
I've also added `@Incubating` to `FailureHandler` new methods and mentioned that `failureFloodingThreshold(long)` is incubating in the docs.